<!---->
<launch>
  <arg name="dir" value="/home/meclab/Desktop/Bag/NCKU_outfield0906/"/>
  <arg name="filename" value="autoware-20220906144103_robosense_ublox_9"/>

  <node name="gnss_csv_filter" pkg="gnss_filter" type="gnss_csv_filter" output="screen"/>
    <param name="directory" value="$(arg dir)"/>
    <param name="filename" value="$(arg filename)"/>
    <param name="filetype" value="_gnss"/>
    <param name="topic" value="/ublox_gps"/>

  <node name="ndt_csv_filter" pkg="gnss_filter" type="ndt_csv_filter" output="screen"/>
  <!-- <node name="gnss_raw_csv_filter" pkg="gnss_filter" type="gnss_raw_csv_filter" output="screen"/> -->
</launch>